### PR TITLE
[dav1d] Update to 0.8.2

### PR DESCRIPTION
**Pull request by EwoutH**
Update [dav1d](https://code.videolan.org/videolan/dav1d) from 0.8.1 to [0.8.2](https://code.videolan.org/videolan/dav1d/-/tags/0.8.2).

### CODE REVIEW

**Review by PhoebeHui — APPROVED** [review]
Thanks for the PR! 

Please ignore the Ci testing failures, I will rerurn this PR after the baseline fixed.

**Review by PhoebeHui — CHANGES_REQUESTED** [review]
@EwoutH, could you update the baseline revision by executing './vcpkg x-add-version dav1d', then submit the changes?

**Comment by EwoutH** [comment]
Done! Feel free to squash when merging.

**Review by PhoebeHui — APPROVED** [review]
Thanks for your updates!

File: ports/dav1d/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "dav1d",
-  "version-string": "0.8.1",
+  "version-string": "0.8.2",
```
Comment (PhoebeHui) [inline ports/dav1d/vcpkg.json:3]:
```suggestion
  "version-semver": "0.8.2",
```

**Review by PhoebeHui — CHANGES_REQUESTED** [review]

**Review by PhoebeHui — APPROVED** [review]

**Pull request merged**